components/prize_pool/commons/prize_pool.lua

```diff
@@ -474,4 +488,17 @@ function Placement:_parseOpponentArgs(input, date)
 	return Opponent.resolve(opponentData, date)
 end
 
+function Placement:_setUsdFromRewards(prizesToUse, prizeTypes)
+	for _, opponent in ipairs(self.opponents) do
+		if not opponent.prizeRewards[PRIZE_TYPE_USD .. 1] then
+			local usdReward = 0
+			for _, prize in pairs(prizesToUse) do
+				local localMoney = opponent.prizeRewards[prize.id] or self.prizeRewards[prize.id] or 0
+				usdReward = usdReward + prizeTypes[prize.type].convertToUsd(prize.data, localMoney, opponent.date)
+			end
+			opponent.prizeRewards[PRIZE_TYPE_USD .. 1] = math.floor(usdReward)
```

Comment:
are we sure we want to use match.floor here?
there are several events that would get way different total prize pools from that
e.g. if €150 are spread to 59 players (yes such events actually exist)
e.g. for https://liquipedia.net/starcraft2/Maralekos_Replay_Liga/4 the usd values with the math.ffloor would only sum up to < 150 instead of ~170
so value in infobox and sum of values in prize pool would hugely differ
(in the above example it would be >11% off)

Reply:
My plan is using rounding to nearest full number, will be changed before ready

Reply:
would maybe 2 digits behind `.` (`1.10`) be something we could consider?

Reply:
Added rounding to 2 decimals